templates/main_template.yml

```diff
@@ -54,7 +53,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          tag: v${{ steps.setup_sp.outputs.plugin-version }}
+          tag: ${{ github.ref_name }}
```

Comment:
If the release tag doesn't match the version present in the .sp, it'll just be an empty release.